### ISSUE TITLE
Plugin ignores custom file found in root directory

### ISSUE THREAD

**Issue by ClaudeHangui** [google/secrets-gradle-plugin#44]
I created a custom file named _service.properties_ at the root of my project and I stored my google api key in it as such: `key_name="value"`
In my module's _build.gradle_ file I have this configuration:
`secrets {
    propertiesFileName 'service.properties'
}`
When I run the project I get the error message saying telling me to check if the api key exist. It does already exists as I originally set my key in the _strings.xml_ file and I read directly from there, before attempting this plugin 
I am using v1.3.0 of the plugin

**Comment by arriolac** [comment]
Hi @ClaudeHangui, can you try with the latest version of this library (2.0.0)? Also, just to confirm, are you able to get the plugin to work when it reads from `local.properties` and not from a custom file? If there are any other error/warning log statements you can share that would be helpful.

**Comment by ClaudeHangui** [comment]
Yes I am able to have it working from the `local.properties` file. We didn't want to upgrade to v2.0.0 as that requires us to move our AGP to v7.x.x
The error logs that we have are as usual i.e when the api key is nor configured or can't be accessed on the project:
`Ensure that the "Google Maps Android API v2" is enabled.
    Ensure that the following Android Key exists:
    	API Key: "my key"`

**Comment by ClaudeHangui** [comment]
The plugin isn't able to fetch the key. Like I said, I initially set up my key in the _strings.xml_ file and I had no issue in displaying my location on the map. I tried the plugin with a custom configuration and file and I have the error above

**Comment by arriolac** [comment]
The error message that you shared in https://github.com/google/secrets-gradle-plugin/issues/44#issuecomment-945976394 does not match the error you are describing of the plugin not being able to fetch the key. 

You should see something like this error for `BuildConfig.GOOGLE_MAPS_KEY` references:

```
Application.kt: (32, 45): Unresolved reference: GOOGLE_MAPS_KEY
```

Or, the following error if the plugin is not able to fetch your key in the AndroidManifest.xml file:

```
Attribute meta-data#com.google.android.geo.API_KEY@value at AndroidManifest.xml:32:13-44 requires a placeholder substitution but no value for <GOOGLE_MAPS_KEY> is provided.
```

Can you share more information from your logs?
